src/Microsoft.OpenApi.Readers/Microsoft.OpenApi.Readers.csproj

```diff
@@ -22,6 +22,13 @@
         <PackageReference Include="SharpYaml" Version="2.1.1" />
     </ItemGroup>
 
+    <!-- Workaround to disable Microsoft.VisualStudio.Threading.Analyzers -->
```

Comment:
I don't think we need to this workaround, we just need to use the `privateAssets` tag with the dependency similar to https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/blob/7312b38e76767f18c1e9ae69e3bc76285d98f1d0/src/Microsoft.Graph.Core/Microsoft.Graph.Core.csproj#L75

https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets